jcl/j9modules.xml

```diff
@@ -83,28 +89,69 @@
 					</else>
 				</if>
 			</case>
+			<case value="jdk.jconsole">
+				<replacerequires module.old="com.ibm.management" module.new="jdk.management" />
+			</case>
+			<case value="jdk.management">
+				<removeexportsoropens action="exports" package="com.sun.management.internal" />
+			</case>
+			<case value="openj9.sfj">
+				<replacerequires module.old="com.ibm.management" module.new="jdk.management" />
+			</case>
 			<default>
 				<echo message="No preparation required for module ${module.name}" />
 			</default>
 		</switch>
 	</target>
 
-	<macrodef name="removeexport">
+	<macrodef name="removeexportsoropens">
 		<attribute name="package" />
+		<attribute name="action" />
 		<sequential>
-			<echo message="Removing export of package @{package}" />
+			<echo message="Removing @{action} of package @{package}" />
 			<loadresource property="regexp.@{package}">
 				<string value="@{package}" />
 				<filterchain>
 					<replaceregex pattern="\." replace="\\\\." flags="g" />
 				</filterchain>
 			</loadresource>
 			<replaceregexp file="${module.dir}/classes/module-info.java" replace="" flags="gs">
-				<regexp pattern="\s*\bexports\s+${regexp.@{package}}\b[^;]*;" />
+				<regexp pattern="\s*\b@{action}\s+${regexp.@{package}}\b[^;]*;" />
 			</replaceregexp>
 		</sequential>
 	</macrodef>
 
+	<macrodef name="replaceexportsoropens">
```

Comment:
When the `module.old` is the only module in `exports|opens` clauses, this `replaceexportsoropens` effectively becomes `removeexportsoropens`, can we combine these two `macrodef` together as well?

Reply:
`removeexportsoropens` removes lines based on the package name being exported while `replaceexportsoropens` uses module names. So `replaceexportsoropens` doesn't remove exports or opens lines that don't have a target module or multiple target modules. 

Should I rename the macro `replaceexportsoropensmodules` for clarity?